### PR TITLE
Remove Polkadot JS mentions from Section Transactions

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK I mostly moved content around. The transaction fee page will probably disappear in January as it has some duplicated content, it can be more concise and put in the transactions page. All JS stuff moved to advanced, mostly in the JS guides.

File: docs/general/attacks.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+id: attacks
+title: Protection from Attacks
```
Comment (profradha) [inline docs/general/attacks.md]: The title is too generic.

Earlier, this section was under "Verifying Extrinsic/Transactions," so it was implicit that these were transaction-related attacks. 

Some titles to consider - 

"Example Attack Scenarios"
"Transaction Verification"
"Why Verify Transactions?"

File: docs/general/attacks.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+sidebar_label: Attack Protection
+description: How to Prevent from being a Victim of an Attack
+keywords: [attack, protection]
+slug: ../attacks
```
Comment (profradha) [inline docs/general/attacks.md]: Same here. An attacks page on Polkadot Wiki could also mean attacks on the protocol, which this page does not cover. Rename this to reflect the narrow scope of transaction based attacks.

File: docs/learn/learn-guides.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+id: learn-guides
+title: Polkadot-JS Guides
+sidebar_label: Polkadot-JS Guides
+description: Guides about Polkadot-JS.
+keywords: [polkadot-JS, polkadotjs, guides, advanced]
+slug: ../learn-guides
```
Comment (profradha) [inline docs/learn/learn-guides.md]: "learn-polkadot-js-guides" would be appropriate slug

**Review by DrW3RK — COMMENTED** [review]
The reorganization makes sense. Some feedback to address, before merging this PR.

File: docs/general/attacks.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+id: attacks
+title: Why Verify Transactions?
+sidebar_label: Transaction Verification
+description: How to Prevent from being a Victim of an Attack
+keywords: [transaction, extrinsic, verification, protection]
+slug: ../attacks
```
Comment (profradha) [inline docs/general/attacks.md]:
```suggestion
slug: ../transaction-attacks
```

File: docs/general/polkadot-vault.md [hunk 5]
```diff
@@ -102,8 +109,8 @@ The procedure to sign transactions with the Vault app is as follows:
 - After scanning the QR code with the Vault app, you will be presented with decoded information
   about what you will sign. Make sure the information matches what you intended to sign in the first
   place. If something does not feel right, do not sign. Check
-  [this page](../learn/learn-extrinsics.md#corrupted-qr-code-parity-signer) for more information and
-  contact [the Polkadot Support Team](https://support.polkadot.network/support/home).
+  [this page](./attacks.md#corrupted-qr-code-parity-signer) for more information and contact
```
Comment (profradha) [inline docs/general/polkadot-vault.md]:
```suggestion
  [this page](./transaction-attacks.md#corrupted-qr-code-parity-signer) for more information and contact
```

File: polkadot-wiki/sidebars.js [hunk 6]
```diff
@@ -148,8 +148,9 @@ module.exports = {
                 slug: '/stay-safe-index',
               },
               items: [
-                "general/scams",
                 "general/how-to-dyor",
+                "general/scams",
+                "general/attacks",
```
Comment (profradha) [inline polkadot-wiki/sidebars.js]:
```suggestion
                "general/transaction-attacks",
```

**Review by DrW3RK — APPROVED** [review]

File: docs/general/attacks.md (hunk 2 above)
Comment (profradha) [reply docs/general/attacks.md]: Renamed the slug to reflect this feedback

**Pull request merged**